pkg/skaffold/kubernetes/log.go

```diff
@@ -91,8 +91,7 @@ func (a *LogAggregator) Start(ctx context.Context, client corev1.CoreV1Interface
 	return nil
 }
 
-// nolint: interfacer
-func (a *LogAggregator) streamLogs(ctx context.Context, client corev1.CoreV1Interface, pod *v1.Pod) error {
+func (a *LogAggregator) streamLogs(ctx context.Context, client corev1.PodsGetter, pod *v1.Pod) error {
```

Comment:
Having this as a corev1interface instead of podsgetter was intentional, even though it failed linting.

Switching it gave me an error downstream. I don't think it would be covered by tests. I'll have to test it again.

Reply:
What do you mean downstream? e2e tests?

Reply:
Yep - i think the error I was thinking of was actually in `wait.go` where I had the `nolint: interfacer`. This LGTM